src/errors.h

```diff
@@ -12,11 +12,11 @@
 #define SW_USER_REJECTED 0x6985
 #define SW_OK            0x9000
 
-#define SW_ASSERT         0x4700
+#define SW_ASSERT        0x4700
 
-
-#define SW_MALFORMED_REQUEST  0x6E01
-#define SW_BAD_CLA            0x6E02
-#define SW_UNKNOWN_INS        0x6E03
+#define SW_MALFORMED_REQUEST          0x6E01
```

Comment:
We will need to sync our two changes, I moved to "enum" style of declaring constants